src/firefly/js/visualize/WebPlotRequest.js

```diff
@@ -1099,7 +1099,17 @@ export class WebPlotRequest extends ServerRequest {
      *
      * @param gridOnStatus GridOnStatus
      */
-    setGridOn(gridOnStatus) { this.setParam(C.GRID_ON, gridOnStatus.key); }
+
+    setGridOn(gridOnStatus= GridOnStatus.FALSE) {
+        var stat= gridOnStatus;
+        if (isBoolean(gridOnStatus)) {
+            stat= gridOnStatus ? GridOnStatus.TRUE : GridOnStatus.FALSE;
+        }
+        else {
+            stat= GridOnStatus.get(gridOnStatus);
+        }
```

Comment:
this took me awhile to figure out what it's doing.
is it just this?
var stat= GridOnStatus.get( String(gridOnStatus) );